packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -54,34 +48,36 @@ function CommentBox({
   errorMessage,
   sendingComment,
   onInput,
-  onKeyDown,
   sendComment,
   parentSelector,
   post,
 }: CommentBoxProps): ReactElement {
-  const onTextareaInput = (content: string) =>
-    onInput(cleanupEmptySpaces(content));
   const { user } = useContext(AuthContext);
-  const commentRef = useRef<HTMLDivElement>(null);
   const {
     onMentionClick,
     onMentionKeypress,
     offset,
     mentions,
     mentionQuery,
     selected,
+    commentRef,
     onInitializeMention,
     onInputClick,
   } = useUserMention({
     postId: post.id,
-    commentRef,
-    onInput: onTextareaInput,
+    onInput,
   });
 
   useEffect(() => {
     commentRef.current?.focus();
-    if (commentRef.current && editContent) {
-      commentRef.current.innerText = editContent;
+    if (commentRef.current) {
+      if (editContent) {
+        commentRef.current.value = editContent;
+      }
+      commentRef.current.setAttribute(
+        'data-min-height',
+        commentRef.current.offsetHeight.toString(),
```

Comment:
Why do we set it as a string when we need to receive it as int anyway?

Reply:
I think this is due to typescript asking it to be a string than a number since attributes are strings. Though let me double check.